examples/cox.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+"""Cox process model using mean-field variational inference.
+   Ref: Miller et al, 2014.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import numpy as np
+import tensorflow as tf
+
+from edward.models import Normal
+from edward.stats import poisson, multivariate_normal_cholesky
+from edward.util import multivariate_rbf
+from scipy.stats import multivariate_normal as mvn, poisson as poi
```

Comment:
I see. I already have the name poisson from edward.stats. But yes this can be updated to reflect your comment.